test-poc/framework/src/main/java/org/keycloak/test/framework/injection/Supplier.java

```diff
@@ -31,4 +31,7 @@ default String getAlias() {
         return getClass().getSimpleName();
     }
 
+    default void onBeforeEach(InstanceContext<T, S> instanceContext) {
```

Comment:
Is it intended it does noting?

Reply:
Majority of suppliers don't need this, it's just for those that need to run something; and for now that's just the event supplier that needs to register the listener after all the managed resources are created

Reply:
Thank you, no I see how robust it is -> available for multiple listeners not necessary Event class.